packages/core/src/git.ts

```diff
@@ -877,6 +877,11 @@ export default class Git {
     return execPromise("git", ["describe", "--tags", "--abbrev=0", since]);
   }
 
+  /** Get the latest released in the git tree with the provided glob pattern */
+  async getLatestReleasedTagWithPattern(pattern?: string) {
```

Comment:
Sorry about that. I included this function to help detect that the `oldRelease` was releasing a maintenance branch with the wrong git tag version. Since we're not using it right now, I will remove it.